### PR TITLE
File selection is required to upload

### PR DESCRIPTION
**Pull request by hmhsu**
### Description
Block upload if no file is selected. Note: it is still possible to skip the upload by clicking the skip button.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
#1210

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/questiontypes/FileUploadQuestionRenderer.java [hunk 1]
```diff
@@ -61,6 +66,13 @@ private ContainerTag signedFileUploadFields(SignedS3UploadRequest request) {
...
         .with(input().withType("hidden").withName("X-Amz-Signature").withValue(request.signature()))
-        .with(input().withType("file").withName("file"));
+        .with(input().withType("file").withName("file"))
+        .with(errorDiv(params.messages()));
+  }
+
+  private ContainerTag errorDiv(Messages messages) {
+    FileUploadQuestion fileuploadQuestion = question.createFileUploadQuestion();
+    return div(fileuploadQuestion.fileRequiredMessage().getMessage(messages))
+        .withClasses(ReferenceClasses.FILEUPLOAD_ERROR, Styles.TEXT_RED_600, Styles.HIDDEN);
```
Comment (kendrickt) [inline universal-application-tool-0.0.1/app/views/questiontypes/FileUploadQuestionRenderer.java:76]: (blocking) there's a `BaseStyles#FORM_ERROR_TEXT_BASE` or `#FORM_ERROR_TEXT_XS`.

Comment (hmhsu) [reply universal-application-tool-0.0.1/app/views/questiontypes/FileUploadQuestionRenderer.java]: Done.

**Review by kendrickt — CHANGES_REQUESTED** [review]
I'm just trying to block this in case you didn't see my slack message. I want to chat before this gets merged in.

**Review by kendrickt — APPROVED** [review]

**Comment by hmhsu** [comment]
> I'm just trying to block this in case you didn't see my slack message. I want to chat before this gets merged in.

Chatted offline. Will have followup PRs to address some UX confusion.

**Pull request merged**